keyboards/clueboard/2x1800/readme.md

```diff
@@ -1,6 +1,11 @@
 # Clueboard 2x1800
 
-Clueboard Double 1800 All The Way
+Clueboard 2x1800
+
+This is a keyboard that Clueboard releases for a 1-day group buy every Apr 1. Each year has a different theme.
+
+* [2018:](2018/) Double 1800 All The Way
```

Comment:
I don't think GitHub's Markdown processing likes having two lists back-to-back.

Lines 7-12 look like five separate lists of one item each, instead of two lists of two and three items, respectively.

Reply:
K, suggestions for this file.

```diff
diff --git a/keyboards/clueboard/2x1800/readme.md b/keyboards/clueboard/2x1800/readme.md
index 01b975695..b88f46837 100644
--- a/keyboards/clueboard/2x1800/readme.md
+++ b/keyboards/clueboard/2x1800/readme.md
@@ -1,18 +1,19 @@
 # Clueboard 2x1800
 
-Clueboard 2x1800
-
 This is a keyboard that Clueboard releases for a 1-day group buy every Apr 1. Each year has a different theme.
 
 * [2018:](2018/) Double 1800 All The Way
 * [2019:](2019/) Mechanical Drawing Toy Edition
 
+## Information
+
 * Keyboard Maintainer: [Zach White](https://github.com/skullydazed)
 * Hardware Supported: Clueboard 2x1800 PCB
-* Hardware Availability: 2018 Apr 1 Group Buy
+* Hardware Availability: Annual Group Buys on April 1
 
-Make example for this keyboard:
+Make examples for this keyboard:
 
-    make 2x1800:default
+    make clueboard/2x1800/2018:default
+    make clueboard/2x1800/2019:default
 
-See [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
```

- Delete Lines 3 and 4 because the line is redundant; there's no relevant photo here. (Should be added to each revision's readme IMO.)
- Split the two lists with a header. Need something here to get proper rendering on GitHub. (This is a work-around for a GitHub issue, really; the existing file isn't *wrong* here.)
- Rephrase the availability. As currently phrased, only time travelers can buy these. :wink: 
- Update `make` examples. Current example doesn't actually work.
- Update the docs links to include the Newbs Guide.